chromadb/api/fastapi.py

```diff
@@ -150,7 +151,13 @@ def _get(
         resp.raise_for_status()
         return resp.json()
 
-    def _delete(self, collection_name, ids=None, where={}, where_document={}):
+    def _delete(
+        self,
+        collection_name: Optional[str],
```

Comment:
I don't think collection_name is Optional

Reply:
Thank you. Fixed.
https://github.com/chroma-core/chroma/pull/200/commits/dcb86f744bc52f2ac81ffdf6222084b0832af76b